### PR TITLE
Add GitSense for GitHub

### PR DESCRIPTION
**Pull request by gitsense**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,8 @@
 - [Github Repo Size](#github-repo-size)
 - [Github Travis Stat](#github-travis-stat)
 - [github.expandinizr](#github.expandinizr)
-- [Gitter Helper](#gitter-helper)
+- [GitSense](#gitsense)
+- [Gitter Helper](#gitsense)
```
Comment (stefanbuck) [inline README.md]: Please revert the changes for gitter helper. Thanks

File: README.md [hunk 2]
```diff
@@ -258,6 +259,15 @@ Add breakpoints at 1400px, 1600px and 1800px for full GitHub experience on large
  ![github.expandinizr](https://dl.dropboxusercontent.com/s/7e9g9g0l445j90m/github-expandinizr.jpg)
 </details>
 
+## [GitSense](https://github.com/gitsense/insight)
+<a href="https://chrome.google.com/webstore/detail/gitsense-insight/fgnjcebdincofoebkahonlphjoiinglo"><img src="https://raw.githubusercontent.com/alrra/browser-logos/master/src/chrome/chrome_48x48.png" width="24" /></a>
+
+Makes searching in GitHub better.  With GitHub, you can only search one branch per non-forked repository. With [GitSense](https://gitsense.com), you search any branch, from any repository, and much more.
```
Comment (stefanbuck) [inline README.md]: Double whitespace after "in GitHub better."

**Comment by gitsense** [comment]
Thanks for merging.
